<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FreeGridGui::SketchParameters</class>
 <widget class="QWidget" name="FreeGridGui::SketchParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sketch size</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="parameters">
     <item row="0" column="0">
      <widget class="QLabel" name="widthLabel">
       <property name="toolTip">
        <string extracomment="Number of 50[mm] units in x direction"/>
       </property>
       <property name="text">
        <string>width</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="sketch_x">
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPushButton" name="inside_sketch_button">
       <property name="text">
        <string>Create sketch</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="validationMessage">
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="sketch_y">
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="depthLabel">
       <property name="toolTip">
        <string extracomment="Number of 50[mm] units in y direction"/>
       </property>
       <property name="text">
        <string>depth</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="sketch_open_front">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Include open front</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
